README.md

```diff
@@ -54,6 +54,16 @@ Please note that some CCLAs require individuals/employees to be explicitly named
 
 *Need an ICLA? Unsure if you are covered under an existing CCLA? Email [help@finos.org](mailto:help@finos.org)*
 
+### Update generated code
+While contributing to the project, you might need to update the generated code.
+Python BDK uses [OpenAPITools/openapi-generator](https://github.com/OpenAPITools/openapi-generator/) to generate code. In order to customise the templates, a fork has been created in [https://github.com/symphony-soufiane/openapi-generator/tree/sym-python-5.4.0](https://github.com/symphony-soufiane/openapi-generator/tree/sym-python-5.4.0).  
```

Comment:
The one to be used is actually from our internal SymphonyPlatformSolutions organisation (https://github.com/SymphonyPlatformSolutions/openapi-generator/tree/sym-python-5.5.0) but it is not accessible for external contributors, so I put my fork we used to use before moving to SymhponyPlatformSolutions.